scripts/monitor.sh

```diff
@@ -51,37 +52,37 @@ while true; do
     check_wifi
 
     # Check if the first program is running.
-    if pgrep -f $PROGRAM_NAME_1 > /dev/null
+    if pgrep -f $PROGRAM_NAMES[0] > /dev/null
     then
         STATUS_1="Online"
     else
         STATUS_1="Offline"
         # If not running, try to restart the program and log its output.
-        $PROGRAM_PATH_1 >> /var/log/programs.log 2>&1 &
+        $PROGRAM_PATHS[0] >> /var/log/programs.log 2>&1 &
         # Store the PID of the restarted program.
         echo $! > /var/run/read_ultralight.pid
     fi
 
     # Check if the second program is running.
-    if pgrep -f $PROGRAM_NAME_2 > /dev/null
+    if pgrep -f $PROGRAM_NAMES[2] > /dev/null
```

Comment:
Owner, so I can't make a suggestion directly, but here is my suggestion:
`if pgrep -f $PROGRAM_NAMES[1] > /dev/null`

Reply:
Thanks for pointing it out!